tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -141,6 +172,9 @@ install_cached_tarball_component() {
 	local current_image_version="${3}"
 	local component_tarball_name="${4}"
 	local component_tarball_path="${5}"
+	# extra_tarballs must be in the following format:
```

Comment:
Looks like you future-proofed this format here? I saw one case where `extra_tarballs` gets set in your new changes (line 380), and it just specifies one tarball (not a space-separated list). Seems reasonable to allow for more in the future, but thought it worth asking.

Reply:
I future proofed this as I can see NVIDIA wanting to also cache their deb files generated from the kernel build.